target/scripts/helper-functions.sh

```diff
@@ -209,6 +209,29 @@ function _monitored_files_checksums
 }
 export -f _monitored_files_checksums
 
+# ? --------------------------------------------- General
+
+function _obtain_hostname_and_domainname
+{
+  if [[ -n "${OVERRIDE_HOSTNAME}" ]]
+  then
+    export HOSTNAME="${OVERRIDE_HOSTNAME}"
+    export DOMAINNAME="${DOMAINNAME:-${HOSTNAME#*.}}"
+    if [[ ! "${DOMAINNAME}" =~ .*\..* ]] # Handle situations where the hostname is name.tld and hostname -d ends up just showing "tld"
+    then
+      DOMAINNAME="${HOSTNAME}"
+    fi
+  else
+    # These hostname commands will fail with "hostname: Name or service not known" if the hostname is not real
```

Comment:
Same here, wrap after 80 chars. This helps me when using split screen in say VS Code to still the whole line without sidescrolling :D

Reply:
> whole line without sidescrolling :D

Enable line wrapping :stuck_out_tongue:

Reply:
> > whole line without sidescrolling :D
> 
> 
> 
> Enable line wrapping :stuck_out_tongue: 

Nah - then I'd have to make out which line are wrapped :D